packages/cli/src/ui/commands/setupGithubCommand.ts

```diff
@@ -0,0 +1,56 @@
+/**
+ * @license
+ * Copyright 2025 Google LLC
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import { execSync } from 'child_process';
+
+import {
+    CommandKind,
+    SlashCommand,
+    SlashCommandActionReturn,
+} from './types.js';
+
+export const setupGithubCommand: SlashCommand = {
+    name: 'setup-github',
+    description: 'Set up GitHub Actions using the Gemini CLI',
+    kind: CommandKind.BUILT_IN,
+    action: (): SlashCommandActionReturn => {
+        const gitRootRepo = execSync('git rev-parse --show-toplevel', {
+            encoding: 'utf-8',
+        }).trim();
+
+        if (!gitRootRepo) {
```

Comment:
Can we conditionally register the command holistically if it's a git repo in BuiltInCommandLoader?

Reply:
Updated, thanks.